### PR TITLE
Opendp pre-release

### PR DESCRIPTION
**Pull request by damienbfs**

### CODE REVIEW

**Review by LancelotMarti — APPROVED** [review]
All good :)

**Comment by PaulineMauryL** [comment]
## Coverage report


 <details><summary>Click to see where and how coverage changed</summary><table><thead>
  <tr><th>File</th><th>Statements</th><th>Missing</th><th>Coverage</th><th>Coverage<br>(new stmts)</th><th>Lines missing</th></tr>
</thead>
<tbody><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/dp_queries/dp_libraries</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb">opendp.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/tests</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c">test_api.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2">test_api_diffprivlib.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-d03a128b76f6654a4b0708c6b8f488eb7fc31178cbfcbec226e4d25bc88150de">test_api_opendp_polars.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-d03a128b76f6654a4b0708c6b8f488eb7fc31178cbfcbec226e4d25bc88150de"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-d03a128b76f6654a4b0708c6b8f488eb7fc31178cbfcbec226e4d25bc88150de"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-d03a128b76f6654a4b0708c6b8f488eb7fc31178cbfcbec226e4d25bc88150de"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-d03a128b76f6654a4b0708c6b8f488eb7fc31178cbfcbec226e4d25bc88150de"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-a4474a6b8ddbabefeb05b2e7282d32a14b08cac2d9ce7a15e7fafc772b2b66e9">test_api_root.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-a4474a6b8ddbabefeb05b2e7282d32a14b08cac2d9ce7a15e7fafc772b2b66e9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-a4474a6b8ddbabefeb05b2e7282d32a14b08cac2d9ce7a15e7fafc772b2b66e9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-a4474a6b8ddbabefeb05b2e7282d32a14b08cac2d9ce7a15e7fafc772b2b66e9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-a4474a6b8ddbabefeb05b2e7282d32a14b08cac2d9ce7a15e7fafc772b2b66e9"></a></td><td><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-a4474a6b8ddbabefeb05b2e7282d32a14b08cac2d9ce7a15e7fafc772b2b66e9R79-R79">79</a></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc">test_mongodb_admin.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/utils</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8f8207a072080f6d0b7cf718ebf9d60f67c21d1d7218facf059a08304a8a8aa9">example_inputs.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8f8207a072080f6d0b7cf718ebf9d60f67c21d1d7218facf059a08304a8a8aa9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8f8207a072080f6d0b7cf718ebf9d60f67c21d1d7218facf059a08304a8a8aa9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8f8207a072080f6d0b7cf718ebf9d60f67c21d1d7218facf059a08304a8a8aa9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-8f8207a072080f6d0b7cf718ebf9d60f67c21d1d7218facf059a08304a8a8aa9"></a></td><td></td></tbody>
<tfoot>
<tr>
<td><b>Project Total</b></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/140/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td>&nbsp;</td>
</tr>
</tfoot>
</table>

<sub>

This report was generated by [python-coverage-comment-action](https://github.com/py-cov-action/python-coverage-comment-action)

</sub>
</details>

File: server/data/collections/dataset_collection_local.yaml [hunk 1]
```diff
@@ -15,14 +15,14 @@ datasets:
   database_type: "S3_DB"
   s3_bucket: "example"
   s3_key: "data/titanic.csv"
-  endpoint_url: "http://minio:9000"
+  endpoint_url: "http://minio:10000"
```
Comment (PaulineMauryL) [inline server/data/collections/dataset_collection_local.yaml:18]: I am curious, why did it need to be modified ?

Comment (LancelotMarti) [reply server/data/collections/dataset_collection_local.yaml:18]: we had conflicts with port 9000 (usually with ipykernel when using jupyter notebook in vscode)

**Pull request merged**